modules/openxr/doc_classes/OpenXRInterface.xml

```diff
@@ -10,6 +10,27 @@
 	<tutorials>
 		<link title="Setting up XR">$DOCS_URL/tutorials/xr/setting_up_xr.html</link>
 	</tutorials>
+	<methods>
+		<method name="get_available_display_refresh_rates" qualifiers="const">
+			<return type="Array" />
+			<description>
+				Returns display refresh rates supported by the current HMD. Only returned if this feature is supported by the OpenXR runtime and after the interface has been initialised.
```

Comment:
Should use en_US spelling so "initiali**z**ed" (same below).

Reply:
I always get that wrong..